tests/e2e/test_pv_scale_ocs_create_delete_pvcs.py

```diff
@@ -138,41 +152,44 @@ def storageclass(self, storageclass_factory):
         self.rbd_sc_obj = storageclass_factory(interface=constants.CEPHBLOCKPOOL)
         self.cephfs_sc_obj = storageclass_factory(interface=constants.CEPHFILESYSTEM)
 
+    # TODO: Skipping memory leak fixture call in test function because of bz 1750328
     def test_pv_scale_out_create_delete_pvcs_with_and_without_io(
-        self, memory_leak_function, namespace, storageclass, setup_fixture, start_io
+        self, namespace, storageclass, setup_fixture, start_io
     ):
-        self.pvc_count_each_itr = 10
-        self.scale_pod_count = 120
-        self.size = '10Gi'
+        pvc_count_each_itr = 10
+        scale_pod_count = 120
+        size = '10Gi'
         test_run_time = 180
         self.all_pvc_obj, self.all_pod_obj = ([] for i in range(2))
         self.delete_pod_count = 0
 
         # Identify median memory value for each worker node
-        median_dict = helpers.get_memory_leak_median_value()
-        log.info(f"Median dict values for memory leak {median_dict}")
+        # TODO: Skipping memory leak median calculate because of bz 1750328
```

Comment:
do we have an issue to track this TODO?

Reply:
If bz 1750328 is resolved then this function works as expected, so there is no separate bug. But requires following enhancements.
1. For now memory leak function runs on all workers but in future we will have worker nodes for OCS and app pods, so function needs enhancement to collect data only on OCS nodes.
2. If a worker nodes has 3 OSD's for now function collects all the ceph-osd daemon stats but needs enhancement in analysis, may be based on process id, but should be intelligent to identify restart of process id(which will change).

Reply:
github issue: https://github.com/red-hat-storage/ocs-ci/issues/811